optimizely/event/processor.go

```diff
@@ -45,14 +45,18 @@ type QueueingEventProcessor struct {
 var pLogger = logging.GetLogger("EventProcessor")
 
 // NewEventProcessor returns a new instance of QueueingEventProcessor with queueSize and flushInterval
-func NewEventProcessor(ctx context.Context, queueSize int, flushInterval time.Duration) *QueueingEventProcessor {
+func NewEventProcessor(ctx context.Context, batchSize, queueSize int, flushInterval time.Duration) *QueueingEventProcessor {
 	p := &QueueingEventProcessor{
 		MaxQueueSize:    queueSize,
 		FlushInterval:   flushInterval,
 		Q:               NewInMemoryQueue(queueSize),
 		EventDispatcher: &HTTPEventDispatcher{},
 	}
 	p.BatchSize = 10
```

Comment:
default batch size.

Reply:
@thomaszurkan-optimizely  using DefaultBatchSize required us to import client package in processor.go which was causing import cycle issue! To avoid that, i have now moved DefaultBatchSize,DefaultEventQueueSize and DefaultEventFlushInterval to processor.go. Will be committing it with some decision service test-cases to improve coverage.

Reply:
@thomaszurkan-optimizely done!